extra/post-ceph-azn.py

```diff
@@ -399,7 +399,7 @@ def add_nova_az(data, num):
                 if data['kind'] == 'OpenStackDataPlaneDeployment':
                     data['spec']['nodeSets'][0] += "-az" + str(args.num)
                 if data['kind'] == 'OpenStackDataPlaneService':
-                    data['spec']['configMaps'][0] += "-az" + str(args.num)
+                    data['spec']['dataSources'][0]['configMapRef']['name'] += "-az" + str(args.num)
```

Comment:
Yes, we need this so it works with https://github.com/openstack-k8s-operators/architecture/pull/296